.ruby-version

```diff
@@ -1 +0,0 @@
-2.5.6
```

Comment:
I'd argue that we don't want to keep this checked in because:

1) CocoaPods and Fastlane support a range of Ruby versions
2) The contents of this file aren't portable across different Ruby version managers, `chruby`, `rbenv`, and `rvm` read this file differently and will break if you are using the wrong one. For example `rbenv` expects `2.5.6` but `chruby` expects `ruby-2.5.6`.

Reply:
Gotcha – point 2 combined with Circle using `chruby` renders the documentary benefits of this moot 😐